packages/medusa/src/api/routes/admin/customers/update-customer.ts

```diff
@@ -64,12 +80,23 @@ export default async (req, res) => {
 
   await customerService.update(id, validated)
 
+  const relations = ["orders"]
+
+  if (validated.groups) {
+    relations.push("groups")
+  }
+
```

Comment:
While I like the idea here I would rather see that we implement an `expand` query param that the client can pass to specify exactly which relations should be joined. See [this](https://github.com/medusajs/medusa/blob/master/packages/medusa/src/api/routes/admin/orders/list-orders.ts) for reference.

What should be possible from the clients perspective is this:

```
POST /admin/customers?expand=groups <-- this will join the groups regardless of whether groups is part of the update payload or not
```

What you will need to do is this:
- Allow `expand` as a query param
- Parse the `expand` fields
- Pass the passed expand fields to relations when retrieving